src/iodata

```diff
@@ -1 +1,6 @@
+#include <QtGlobal>
```

Comment:
Hey @neochapay this import doesn't actually exist right now. Instead (for Qt5) this would be `qt5/QtCore/QtGlobal`. Is some configuration missing in the .pro files to make this work?